### PR TITLE
v5.26 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
@mkraft Please help with the following details for v5.26 changelog:
1. Any new config.json settings.
2. Any Database changes.
3. Any Websocket event changes.
4. Any API changes.

**Comment by amyblais** [comment]
@mkraft Kind reminder for https://github.com/mattermost/docs/pull/3757#issuecomment-668576498.

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,129 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Fixed an issue where a highlight was missing when users at-mentioned themselves, followed by period, underscore, or hyphen.
+ - Fixed an issue where a 500 error was returned by the ``/posts/unread`` endpoint caused by an [integer overflow](https://en.wikipedia.org/wiki/Integer_overflow) when ``limit_after`` was set to 0.
+ - Fixed an issue where the footer text in invitation emails was not translated.
+ - Fixed an issue where ``PermanentDeleteTeam`` did not return an error but did a soft deletion if ``EnableAPITeamDeletion`` was not set.
+ - Fixed an issue on PostgreSQL where logging in using MFA did not respect the uppercase of the email address.
+
+### config.json
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
```
Comment (amyblais) [inline source/administration/changelog.md:99]: Config settings, API, Database, Websocket info will come from Martin

**Comment by mkraft** [comment]
> @mkraft Please help with the following details for v5.26 changelog:
> 
> 1. Any new config.json settings.
> 2. Any Database changes.
> 3. Any Websocket event changes.
> 4. Any API changes.

# Config changes
* Adds `ServiceSettings.ExperimentalDataPrefetch`
* Adds `ClusterSettings.EnableExperimentalGossipEncryption`
* Adds `LogSettings.EnableSentry`
* Adds `LogSettings.AdvancedLoggingConfig`
* Adds `FileSettings.AmazonS3PathPrefix`
* Adds `EmailSettings.PushNotificationBuffer`
* Adds `SupportSettings.EnableAskCommunityLink`
* Adds `GlobalRelayMessageExportSettings.SMTPServerTimeout`

# Database
* Adds column `Sessions.ExpiredNotify`

# Websockets
* Adds event `sidebar_category_created`
* Adds event `sidebar_category_updated`
* Adds event `sidebar_category_deleted`
* Adds event `sidebar_category_order_updated`
* Adds event `warn_metric_status_received`
* Adds event `warn_metric_status_removed`

# API changes
* New route `POST api/v4/bots/:bot_id/convert_to_user`
* New route `GET api/v4/teams/:team_id/channels/private`

**Pull request merged**